README.md

```diff
@@ -546,7 +546,7 @@ interface {
 }
 ```
 
-#### [Check whether a given Command Classes API is supported by the above method]
```

Comment:
this was meant to link to https://zwave-js.github.io/node-zwave-js/#/api/endpoint?id=supportsccapi but I forgot to add the link. Can we revert and add the link in ()?